### ISSUE TITLE
Should we stop testing ZEO against all ZODB storage types?

### ISSUE THREAD

**Issue by d-maurer** [zopefoundation/ZEO#198]
Running the ZEO test suite in a single variant takes (on my local system) between 7 (py3.x) and 10 (pypy 3.x) minutes. I find this quite long.

A major cause for the long time: the tests check ZEO for all ZODB storage types, e.g. for `FileStorage`, for `HexFileStorage`, for `MappingStorage`, for `DemoStorage`, ...
I doubt that this is necessary: we are testing ZEO, not the storages. If ZEO is tested against a model storage, ZEO should work too with other well behaving storages.
In my view, it should be sufficient to test ZEO with `FileStorage` with and without blobs.

Opinions?

**Comment by dataflake** [comment]
I agree. Should a ZEO test against a specific storage fail but succeed with another storage then that's a problem with a storage implementation, not ZEO itself. Obviously FileStorage, being the "storage of record" for ZODB, is the best test candidate.

**Comment by icemac** [comment]
Sounds reasonable to me.
